public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -49,6 +50,32 @@ export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.E
     );
   }
 
+  if (config.featureToggles.externalServiceAccounts && pageId === PluginTabIds.IAM) {
+    return (
+      <Stack direction="column">
+        <Stack direction="row">The {plugin.name} plugin requires the following permissions on your instance:</Stack>
+        <Stack direction="row">
+          <table className="filter-table">
+            <thead>
+              <tr>
+                <th className="admin-settings-section">Action</th>
+                <th className="admin-settings-section">Scope</th>
+              </tr>
+            </thead>
+            <tbody>
+              {plugin.iam?.permissions.map((permission: Permission, i: number) => (
```

Comment:
I think it can be `plugin.iam?.permissions` from local plugin or `plugin.details?.iam?.permissions` for a fetched plugin?